sdk/nodejs/runtime/template/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "template",
+  "version": "1.0.0",
+  "main": "index.js",
+  "license": "MIT",
+  "dependencies": {
+    "tsx": "^4.6.2",
```

Comment:
If you have a global tsx, why put it here?

Reply:
Because I execute the `tsx` binary in the entrypoint, not the on downloaded as dependency so if the user update the package.json to remove it, it will still work :D

Reply:
But you're not supposed to run modules in the host. It should always go through the runtime.

Reply:
Fixed in [a1c61d9](https://github.com/dagger/dagger/pull/6220/commits/a1c61d968be22f8dee5d74b590d8938a6d0c9fb6)